html/changelogs/purplepineapple-PR-15681.yml

```diff
@@ -0,0 +1,7 @@
+author: PurplePineapple
```

Comment:
Add the spriter to the author section as well.

Reply:
Oh forgot to. Thanks for the reminder